src/com/google/enterprise/adaptor/SensitiveValueCodec.java

```diff
@@ -298,15 +317,33 @@ public static void main(String[] args) throws IOException {
       }
     }
     SensitiveValueCodec codec = new SensitiveValueCodec(keyPair);
-    
-    Console console = System.console();
-    if (console == null) {
-      log.warning("Couldn't get Console instance");
-      return;
+    boolean quietParameterPresent = isParameterPresent(args, "--quiet");
+    if (quietParameterPresent) {
+      BufferedReader reader = new BufferedReader(new InputStreamReader(
+              System.in));
+      String encodedValue = codec.encodeValue(reader.readLine(),
+              securityLevel);
```

Comment:
make the indent on this line and BufferedReader one be 4spaces rather than 8, please